src/components/Shifting/ShiftDetails.tsx

```diff
@@ -5,8 +5,12 @@ import { useState } from "react";
 import { CopyToClipboard } from "react-copy-to-clipboard";
 import { useTranslation } from "react-i18next";
 
+import Card from "@/CAREUI/display/Card";
 import RecordMeta from "@/CAREUI/display/RecordMeta";
 import CareIcon from "@/CAREUI/icons/CareIcon";
+import PrintPreview from "@/CAREUI/misc/PrintPreview";
+
+import { CardContent, CardHeader, CardTitle } from "@/components/ui/card";
```

Comment:
Use shadcn-ui's card instead. Let's not mix both.

Reply:
@rithviknishad  updated